exasol/ai/mcp/server/mcp_server.py

```diff
@@ -271,3 +274,24 @@ def execute_query(
             result = self.connection.execute_query(query, snapshot=False).fetchall()
             return ExaDbResult(result)
         raise ValueError("The query is invalid or not a SELECT statement.")
+
+    async def execute_write_query(
+        self, query: Annotated[str, Field(description="DML or DDL query")], ctx: Context
+    ) -> None:
+        if not self.config.enable_write_query:
+            raise RuntimeError(
+                "The execution of Data Definition and "
+                "Data Manipulation queries is disabled."
+            )
+        confirmation = await ctx.elicit(
+            message=(
+                "The following Data Definition or Data Manipulation query will be executed "
+                f"if permitted. Please accept or decline the query execution.\n\n{query}"
```

Comment:
```suggestion
                f"if permitted. Please check the query carefully if it could cause dangerous changes and accept or decline the query execution.\n\n{query}"
```